api/v1alpha1/dspipeline_types.go

```diff
@@ -35,6 +35,9 @@ type DSPASpec struct {
 	*MlPipelineUI `json:"mlpipelineUI"`
 	// +kubebuilder:validation:Required
 	*ObjectStorage `json:"objectStorage"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:={deploy: false}
+	*MLMD `json:"mlmd"`
```

Comment:
Can you update the docs on how one can enable MLMD: 
https://github.com/opendatahub-io/data-science-pipelines-operator